src/Public.Api/Road/Uploads/UploadController-Post.cs

```diff
@@ -0,0 +1,44 @@
+namespace Public.Api.Road.Uploads
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Common.Infrastructure;
+    using Infrastructure;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+
+    public partial class UploadController
+    {
+        [HttpPost("wegen/upload")]
+        [DisableRequestSizeLimit]
+        [RequestFormLimits(MultipartBodyLengthLimit = int.MaxValue, ValueLengthLimit = int.MaxValue)]
```

Comment:
Is there a reason why this is included here and not on `ExtractController-PostUpload.cs` ?

Reply:
Because it's also used on the corresponding endpoint in the [road registry](https://github.com/Informatievlaanderen/road-registry/blob/d03913a1531efc40fdba848ed1a9e721b16969c0/src/RoadRegistry.BackOffice.Api/Uploads/UploadController.cs#L39), while that's not the case for `ExtractController-PostUpload.cs`. I can apply it if needed?

Reply:
Nah, it's okay.